packages/introspect/src/generate.ts

```diff
@@ -186,12 +216,14 @@ function createRootJsonSchema(
     {
         defaultSchema,
         includeTables,
+        schemaDefOverrides,
     }: {
         defaultSchema: string;
         includeTables: string[];
+        schemaDefOverrides: SchemaDefOverrides | undefined;
```

Comment:
instead of passing undefined around all the time, you can just `?? {}` at the top level.